internal/impl/writer.go

```diff
@@ -0,0 +1,37 @@
+package impl
+
+import (
+	"io"
+	"strings"
+)
+
+var nixPackageInstallIgnore = []string{
+	`replacing old 'devbox-development'`,
+	`installing 'devbox-development'`,
+}
+
+type nixPackageInstallWriter struct {
+	w io.Writer
+}
+
+func (fw *nixPackageInstallWriter) Write(p []byte) (n int, err error) {
+	lines := strings.Split(string(p), "\n")
+	for _, line := range lines {
+		if line != "" && !fw.ignore(line) {
+			_, err = io.WriteString(fw.w, "\t"+line+"\n")
+			if err != nil {
+				return
+			}
+		}
+	}
```

Comment:
didn't know the indentation would be so involved!

Reply:
There may be a simpler way? But we need to make sure to write strings as they are being outputted, we can't really wait for the full string and add later.